### ISSUE TITLE
Train new data

### ISSUE THREAD

**Issue by Kik099** [huckiyang/speech_quantum_dl#7]
How can I train different words?

**Comment by huckiyang** [comment]
Hi, first you need some basic idea of CTC Loss [A].

0. This discussion provides a good implementation example. 
https://stackoverflow.com/questions/57292896/understanding-ctc-loss-for-speech-recognition-in-keras

If you want to train in different words or languages modified from this repo,
1. You need to extra Mel-features (with QCNN or CNN) and give each audio with a corresponding text label, e.g., "oui" and "non" for French audio commands.
2.  Replace the label in this line (https://github.com/huckiyang/QuantumSpeech-QCNN/blob/main/qsr_ctc_wer.py#L18)
3.  Modify the Maximum Text length (https://github.com/huckiyang/QuantumSpeech-QCNN/blob/main/qsr_ctc_wer.py#L36) OR using a Flexible Length setting in [0]. It might take some engineering time for a flexible length tf 2.0 layer. 

Hope these suggestions help your interest. 

A. Graves, Alex; Fernández, Santiago; Gomez, Faustino (2006). "Connectionist temporal classification: Labelling unsegmented sequence data with recurrent neural networks". In Proceedings of the International Conference on Machine Learning, ICML 2006
